libraries/AP_RangeFinder/AP_RangeFinder_Ainstein_LR_D1.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "AP_RangeFinder_config.h"
+
+/*
+ questions:
+ - bug on page 22 on malfunction codes
+ - fixed length seems strange at 28 bytes
+ - definition of snr field is missing in documentation
+ - roll/pitch limits are in conflict, 3.2 vs 
+*/
+
+#include "AP_RangeFinder.h"
+#include "AP_RangeFinder_Backend_Serial.h"
+
+class AP_RangeFinder_Ainstein_LR_D1 : public AP_RangeFinder_Backend_Serial
+{
+
+public:
+
+    static AP_RangeFinder_Backend_Serial *create(
+        RangeFinder::RangeFinder_State &_state,
+        AP_RangeFinder_Params &_params) {
+        return new AP_RangeFinder_Ainstein_LR_D1(_state, _params);
+    }
+
+protected:
+
+    MAV_DISTANCE_SENSOR _get_mav_distance_sensor_type() const override {
+        return MAV_DISTANCE_SENSOR_RADAR;
+    }
+
+    uint32_t initial_baudrate(uint8_t serial_instance) const override {
```

Comment:
Why is this needed? Can the baudrate of the sensor be changed? If yes, then that can be handled by SERIALx_BAUD